src/Console/CommandInterface.php

```diff
@@ -49,6 +49,14 @@ interface CommandInterface
      */
     public function setName(string $name);
 
+    /**
+     * Set the description of this command
+     *
+     * @param string $description The description for the command
+     * @return mixed
+     */
+    public function setDescription(string $description);
```

Comment:
I am not sure if adding to the interface is OK for a minor.
Usually we just add those as annotations to the class to make STAN happy, clarify the method existence and provide some sort of soft contracting.

Reply:
Right, I thought I just mimicked the name property setup present but actually that function is not used. I will remove that.